test/e2e/common/utils.py

```diff
@@ -46,3 +46,24 @@ def predict(service_name, input_json):
         logging.info("Got response code %s, content %s", response.status_code, response.content)
         probs = json.loads(response.content.decode('utf-8'))["predictions"]
         return probs
+
+def explain(service_name, input_json):
+    isvc = KFServing.get(service_name, namespace=KFSERVING_TEST_NAMESPACE)
+    # temporary sleep until this is fixed https://github.com/kubeflow/kfserving/issues/604
+    time.sleep(10)
```

Comment:
Is this enough? If there's a race, I'd like to overkill it so we never see it. 1 second okay?

Reply:
@ellis-bigelow 
Thanks for your review feedback. Please correct me if I have missed something here. The current code [sleep](https://docs.python.org/3/library/time.html?highlight=time#time.sleep) for 10 sec, are you saying to make it sleep longer, like 15 sec?

Reply:
For some reason I thought this was milliseconds. You're all over it!